docker/app/utils.py

```
@@ -1,40 +1,360 @@
+import wandb
+from layers import PositionalEncoding
+
+# 評価指標用のダウンロード
+def download_nltk_resources():
+    # nltk_dataディレクトリがない場合のみダウンロード
+    resources = ['punkt', 'wordnet', 'omw-1.4']
+    for resource in resources:
+        if not os.path.exists(os.path.join(os.path.expanduser("~"), "nltk_data", resource)):
+            try:
+                nltk.download(resource)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)